packages/bits/src/services/unit-conversion.service.ts

```diff
@@ -1,16 +1,31 @@
 import { Injectable } from "@angular/core";
 
+import { UnitBase, unitConversionConstants, UnitOption } from "../constants";
+
 import { LoggerService } from "./log-service";
 import { IUnitConversionResult } from "./public-api";
 
 /**
- * @ignore
+ * <example-url>./../examples/index.html#/common/unit-conversion-service</example-url>
+ */
+
+/**
+ * Service for converting a raw value to a larger unit approximation of the value--for example, 1024 B to 1 MB, 12345 Hz to 12.35 kHz, etc.
```

Comment:
Shouldn't we have spaces between words? Instead of "value--for" -> "value - for".

Reply:
Good question. I'm using `--` as a crude approximation of an [em dash](https://www.thesaurus.com/e/grammar/em-dash/).